src/help/zaphelp/contents/ui/dialogs/options/checkforupdates.html

```diff
@@ -46,8 +46,8 @@
 They may be incomplete, contain significant issues or cause stability problems.
 
 <H3>Add-on directories</H3>
-ZAP will load all of the add-ons located in the release 'plugins' directory as well as any that have been downloaded
-and are in the local 'plugins' directory.<br>
+ZAP will load all of the add-ons located in the release 'plugin' directory as well as any that have been downloaded
+and are in the local 'plugin' directory.<br>
```

Comment:
Yeah, I was wondering that, maybe remove/tweak this paragraph to point to the other page (to have that in just one place)?
I was also thinking on changing that to use the names used in the FAQs (https://github.com/zaproxy/zaproxy/wiki/FAQconfig and https://github.com/zaproxy/zaproxy/wiki/FAQinstall), default and install?